src/einsteinpy/bodies/body.py

```diff
@@ -0,0 +1,43 @@
+import astropy.constants as const
+import astropy.units as u
+import numpy as np
+
+
+class Body:
```

Comment:
Can we have two classmethods for the Body class?

Reply:
I have updated the class and added the classmethods.
Will update the PR after some testing.